specification/confidentialledger/data-plane/readme.python.md

```diff
@@ -8,15 +8,16 @@ python:
   clear-output-folder: true
   license-header: MICROSOFT_MIT_NO_VERSION
   no-namespace-folders: true
-  package-version: "2022-20-04"
+  package-version: 0.1
```

Comment:
@lmazuel If Confidential Ledger is using our new DPG process (I think it is), then shouldn't the Python autorest configuration be in the azure-sdk-for-python repo and not here?